### DISCUSSION TITLE
Fetch records 20 records per request  with click in pagination

### DISCUSSION THREAD

**Discussion by mm0hammadi** [KevinVandy/material-react-table#375] [Q&A]
I want to fetch 20 records from the database  and backEnd  every time the pagination  is clicked  
how can i do it?

**Comment by KevinVandy** [comment]
Have you started by looking at either the [Remote Fetching Example](https://www.material-react-table.com/docs/examples/remote) or the[ React Query Example](https://www.material-react-table.com/docs/examples/react-query)? The [Pagination docs](https://www.material-react-table.com/docs/guides/pagination#manage-pagination-state) themselves cover a lot about this too

**Reply by mm0hammadi** [reply]
i useing remote fetching  and useing redux toolkit for passing data to data props
